sensorprocess/sensorprocess_test.go

```diff
@@ -17,43 +20,22 @@ import (
 	"github.com/viamrobotics/viam-cartographer/sensors/inject"
 )
 
-type addLidarReadingArgs struct {
```

Comment:
This file was starting to get big and difficult to read, so I moved out all the helper functions & co into `sensorprocess/testhelper.go`.